src/decoder/window-decoder.ts

```diff
@@ -118,4 +121,22 @@ export default class WindowDecoder extends Decoder {
       this.resetAudioDecoder();
     }
   }
+
+  private async onMPEG1AudioEmitted({ pts_timestamp, data }: Events[typeof EventTypes.MPEG1AUDIO_EMITTED]) {
+    const encodedAudioChunk = new EncodedAudioChunk({
+      type: 'key',
+      timestamp: pts_timestamp * 1000000,
```

Comment:
What's the magic constant? @jirizalud

Reply:
As far as I know pts_timestamp is obtained from "video_pts_elapsed_seconds" but standard timestamp should be in microseconds, so it is a conversion constant from s -> us